.chloggen/fix-file-sort-layout-overwrite.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: "bug_fix"
```

Comment:
I don't think we should have a changelog entry for this since it addresses a change that hasn't been included in a release

Reply:
Removed